dask/dataframe/shuffle.py

```diff
@@ -208,7 +211,7 @@ def set_partition(
             column_dtype=df.columns.dtype,
         )
 
-    df4.divisions = divisions
+    df4.divisions = divisions.tolist()  # [v for v in divisions.values]
```

Comment:
Remove the comment? Or clarify it, since it's not obvious (to me) what value it's adding.

Reply:
Sorry - this was a mistake.  Should be `[v for v in divisions.values]`, since cudf does not currently have a `tolist`

Reply:
Ah, understood.

I worry a bit about this change. pandas `.tolist()` also handles converting from NumPy types to the equivalent Python ones.

```python
In [16]: s = pd.Series([1, 2, 3])

In [17]: type(list(s.values)[0])
Out[17]: numpy.int64

In [18]: type(s.tolist()[0])
Out[18]: int
```

Can you verify that this still ends up with the same scalar types? I don't quite understand what the old code was doing, so this may be a moot point.

Reply:
Sorry - you're right.  This was supposed to be `[v for v in divisions]`